src/services/client.service.ts

```diff
@@ -0,0 +1,25 @@
+import { EpicGamesMapperHelper } from "../helpers/mappers/epic-games.mapper";
+import { ElementToSendInterface, EpicGamesDatasInterface } from "../interfaces/client.interface";
+import fs from "fs";
+
+export class ClientService {
+
+    /**
+     * Update `cache.json` with fresh datas from Epic Games
+     * @param {EpicGamesDatasInterface} data Datas from Epic Games
+     */
+    public async updateCache(data: EpicGamesDatasInterface): Promise<void> {
+        try {
+            const elementsToSave: ElementToSendInterface[] = EpicGamesMapperHelper.map(data);
+            
+            if (elementsToSave.length) {
+                fs.writeFile("src/data/cache.json", JSON.stringify(elementsToSave, null, 4), () => {
+                    console.log("Cache Updated !");
```

Comment:
Same logic to be more explicit and relevant:
```suggestion
                    console.log("EpicGames cache file was updated");
```

Reply:
Done